flexget/plugins/filter/exists_series.py

```diff
@@ -88,7 +90,9 @@ def on_task_filter(self, task, config):
                     # run parser on filename data
                     try:
                         disk_parser = get_plugin_by_name('parsing').instance.parse_series(data=filename.name,
-                                                                                          name=series_parser.name)
+                                                                                          name=series_parser.name,
```

Comment:
This should be indented to just past the parenthesis in order to be pep8-compliant.